common/config/environment.go

```diff
@@ -272,3 +279,13 @@ type Environment interface {
 func (e *CommonEnvironment) TestingWorkloadDeploy() bool {
 	return e.GetBoolWithDefault(e.TestingWorkloadConfig, DDTestingWorkloadDeployParamName, true)
 }
+
+// Dogstatsd namespace
+
+func (e *CommonEnvironment) DogstatsdDeploy() bool {
+	return e.GetBoolWithDefault(e.DogstatsdConfig, DDDogstatsdDeployParamName, false)
```

Comment:
For all the other types of workload, the `deploy` option defaults to `true`.

Cons of having all `deploy` options defaulting to `true`:
* By default and without any explicit parameter from the user, we get the fullest setup.
  * It could take the more time to startup but as everything can be started in parallel, I’m not sure it’s that true.
  * It could be more expensive but as long as everything fits on the same number of k8s nodes, it doesn’t change anything.

Pros of having all `deploy` options defaulting to `true`:
* By default and without any explicit parameter from the user, they will test everything. So, it highly reduces the risk of having people committing things that haven’t been tested.

```suggestion
	return e.GetBoolWithDefault(e.DogstatsdConfig, DDDogstatsdDeployParamName, true)
```

Reply:
Right, at first I set it to false because of the conflicts with the agent, but now it should be OK.
I think we can set it to to true. I don't think the dogstatsd-standalone deployment takes a lot of time compared to everything else that needs to be deployed.